src/utils/config-helper.ts

```diff
@@ -44,6 +44,8 @@ export function fetchConfig (): Promise<any> {
       sessionStorage.setItem('AUTH_API_URL', authApiUrl)
       console.log('Set Auth API URL to: ' + authApiUrl)
 
+      sessionStorage.setItem('VUE_APP_AUTH_ROOT_API', response.data['VUE_APP_AUTH_ROOT_API'])
```

Comment:
I think you can use `AUTH_API_URL` instead.

Reply:
Same as above - we'll look at renaming as a part of a tech-debt ticket in the near future

Reply:
A new ticket has been created for this - Entity #2960